src/Troubleshooting/index.md

```diff
@@ -8,6 +8,79 @@ This is due to a Roblox update causing an incompatibility with Wayland's handlin
 
 #### White Screen
 
-This specific problem can be the same as many other root causes, a common one is your graphics card not supporting Vulkan, or is missing the graphics libraries required for Wine; you can see [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md)
+This specific problem can be the same as many other root causes, a common one is your system missing the graphics libraries required for Wine; you can check [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) out for more information.
 
-This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
+This may also indicate your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
+
+#### Make sure there are no stale wineservers running
+
+There are multiple reasons why this'd occur, but `vinegar kill` will suffice in order to fix the issue.
+
+#### Roblox Studio stuck on the splash
+
+This usually happens due to issues with the DPI. In order to fix it, set your DPI to 97 by running `vinegar exec winecfg` to open the wine configuration, and then heading to the Graphics tab. You should be able to find a DPI option at the bottom of the page.
+
+
+The only downside of this is that the Roblox fonts will be a bit blurry. You could try setting it to an even number larger than 96, but this'd cause in a bigger interface within wine.
```

Comment:
```suggestion
A DPI greater than 96 could be set, but this causes in-game GUIs to be larger than normal. Also, because of a Wine issue, using a non-default DPI will cause the Roblox viewport to be a bit blurry. Additionally, make sure that the DPI is an even number, otherwise GUIs will be blurry.
```